src/input-group.scss

```diff
@@ -60,7 +60,8 @@ $fd-input-border-radius: var(--sapField_BorderCornerRadius);
     @include fd-reset-child-spacing();
 
     border: none;
-    flex: 1;
+    flex: 1 1 auto;
+    width: auto;
```

Comment:
Do we need the `width: auto` when we have `flex: 1 1 auto`?

Reply:
apparently not...fixed